featuretools/utils/wrangle.py

```diff
@@ -100,17 +100,16 @@ def _check_time_against_column(time, time_column):
     return False
 
 
-# --> TODO possibly remove this or convert to Woodwork
 def _check_time_type(time):
     '''
     Checks if `time` is an instance of common int, float, or datetime types.
     Returns "numeric", "datetime", or "unknown" based on results
```

Comment:
Not a big deal since this is a private function, but we probably should update the docstring here to match the code.